### ISSUE TITLE
Yuzu runs at Half Speed When Streaming via Steam Link

### ISSUE THREAD

**Issue by FurbyTime** [yuzu-emu/yuzu#7335]
When streaming any Yuzu game over Steam Link, Yuzu performs at half speed (i.e. 30 fps games run at 15, 60 at 30, etc). Audio Performance (Music, sound effects, etc) appear to be unaffected- Music runs at proper speed and audio effects are not stretched.

Curiously, this behavior actually STOPS if the system is accessed via a remote software system once while it is running (In this case, Chrome Remote Desktop), and the game will then run at it's correct speed for the remainder of the time that instance of Yuzu is active, even once the remote desktop connection has been closed.

Yuzu Version: Yuzu 808 (2021-11-13)
Graphics Settings: Vulkan (Also observed with an OpenGL configuration), "Docked".
Games Observed in: The Legend of Zelda: Link's Awakening, The Legend of Zelda: Breath of the Wild, Super Mario Odyssey

System: Windows 10 Pro
Version 21H1, Build 19043.1348
CPU: AMD Ryzen 5950x
GPU: Nvidia RTX 2080 TI

**Comment by G-Spawn** [comment]
Idk if you can replicate this easily but for me I always get best result opening steam link to device into big picture vs launching direct game (eg linking a pi with steamlink to my pc and it opens big picture) and then when close big picture it drops me to a glorified remote desktop connection at that point and you can launch whatever you want (yuzu included) at much better latency than typical steamlink connection for me personally. Same can be done on phone iirc but it has been a minute since I've tried this.

**Comment by yodaut** [comment]
i had a very similar problem and it was related to g-sync. I have an nvidia GPU as well and a g-sync capable monitor. when i used the nvidia control panel to force the "monitor technology" to "fixed refresh" as a custom 3D program setting for yuzu.exe, the issue went away. something to try out if your configuration has g-sync displays in the mix.
originally mentioned here: https://www.reddit.com/r/yuzu/comments/rxiygl/yuzu_starts_with_lag_and_very_low_fps_but_returns/

**Comment by FurbyTime** [comment]
> Idk if you can replicate this easily but for me I always get best result opening steam link to device into big picture vs launching direct game (eg linking a pi with steamlink to my pc and it opens big picture) and then when you close big picture it drops me to a glorified remote desktop connection at that point and you can launch whatever you want (yuzu included) at much better latency than typical steamlink connection for me personally. Same can be done on phone iirc but it has been a minute since I've tried this.

This one's not quite a lag or connection problem, though - I've moved onto Moonlight since then, and between doing a similar connection to what you listed as well as using it's version of Big Picture Mode still produces the same result. It's something related to Yuzu itself.

>i had a very similar problem and it was related to g-sync. I have an nvidia GPU as well and a g-sync capable monitor. when i used the nvidia control panel to force the "monitor technology" to "fixed refresh" as a custom 3D program setting for yuzu.exe, the issue went away. something to try out if your configuration has g-sync displays in the mix.

Wow, that actually did it! Thanks, that saves me a lot of confusion when starting a game!

I've gone ahead and renamed the topic so, if this is a dev focus, they can address it. It doesn't have anything to do with remote software, but G-Sync.

**Comment by Hexaae** [comment]
Yep, with G-Sync on Yuzu seems to run at half speed with many games. I use NVidia Profile Inspector with _GSYNC - Application state: Disallow_ as a temporary workaround.
